src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -76,8 +76,9 @@ class EnsembleFlowCustomNodePipelineExecutionTest : public TestWithTempDir {
     template <typename T>
     std::unique_ptr<Pipeline> prepareSingleNodePipelineWithLibraryMock() {
         const std::vector<float> inputValues{3.5, 2.1, -0.2};
+        const tensor_map_t inputsInfo{{pipelineInputName, nullptr}};
```

Comment:
Why does this work with uninitialized tensorinfo? (nullptr)

Reply:
Because it's not used in that scenario. `inputsInfo` is necessary to fill new EntryNode constructor and it's used only for binary inputs right now.